src/assets/css/fonts.scss

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "fontHelvetica";
+  src: url("https://pushdev-code.github.io/shoping-cart/public/fonts/helveticaneue_medium-webfont.woff2");
```

Comment:
since you're using the fonts from a URL, don't include the font files in this PR.

Reply:
Done